src/run/parser/livesplit.rs

```diff
@@ -433,7 +433,7 @@ fn parse_attempt_history(version: Version, reader: &mut Reader<'_>, run: &mut Ru
 pub fn parse(source: &str, path: Option<PathBuf>) -> Result<Run> {
     let reader = &mut Reader::new(source);
 
-    let mut image_buf = Vec::with_capacity(4096);
+    let mut image_buf = Vec::new();
```

Comment:
I also snuck this in. Nowadays we know exactly how much memory we need for each image in here when we encounter it, so there's no need to preallocate anything if we are going to reallocate the memory anyway the moment we encounter an image.